core/src/test/scala/io/projectglow/transformers/blockvariantsandsamples/BlockVariantsAndSamplesTransformerSuite.scala

```diff
@@ -150,4 +153,18 @@ class BlockVariantsAndSamplesTransformerSuite extends GlowBaseTest with GlowLogg
         assert(SQLUtils.structFieldsEqualExceptNullability(t, e), s"Expected\n$e\nBlocked\n$t")
     }
   }
+
+  test("inconsistent number of values") {
+    val vcfDf = spark
+      .read
+      .format("vcf")
+      .load(testVcf)
+      .withColumn("values", expr("array_repeat(0.0, INFO_DP)"))
+    val options = Map(VARIANTS_PER_BLOCK -> "10", SAMPLE_BLOCK_COUNT -> "20")
+    val ex = intercept[SparkException] {
+      Glow.transform(TRANSFORMER_NAME, vcfDf, options).show()
+    }
+    assert(ex.getCause.isInstanceOf[RuntimeException])
+    assert(ex.getCause.getMessage.contains("is not true"))
```

Comment:
What does the error message look like here?

Reply:
It looks like this: ```java.lang.RuntimeException: '(size(array_repeat(0.0, input[4, int, true]), true) = 17190)' is not true!```

Reply:
Hm, that's an inscrutable error message. If we give the column an informative name, does the alias appear in the error message?

Reply:
The column is already called `values`; maybe I can re-throw with a more helpful message.

Reply:
Actually, I'm not sure if we can re-throw in this case as the issue doesn't manifest until the DataFrame is materialized.